l10nzh.properties

```diff
@@ -1,18 +1,30 @@
 category=Localization
 description=SonarQube Chinese Pack
 homepageUrl=https://github.com/SonarCommunity/sonar-l10n-zh
-archivedVersions=
-publicVersions=9.9,10.0,10.1,10.2,10.3,10.4
+archivedVersions=10.4
+publicVersions=9.9,10.0,10.1,10.2,10.3,10.4.1,10.5
 
 defaults.mavenGroupId=org.codehaus.sonar-plugins.l10n
 defaults.mavenArtifactId=sonar-l10n-zh-plugin
 
+10.5.description=Support SonarQube 10.5
+10.5.sqVersions=[10.5,LATEST]
+10.5.date=2024-04-13
+10.5.changelogUrl=https://github.com/xuhuisheng/sonar-l10n-zh/releases/tag/sonar-l10n-zh-plugin-10.5
+10.5.downloadUrl=https://github.com/xuhuisheng/sonar-l10n-zh/releases/download/sonar-l10n-zh-plugin-10.5/sonar-l10n-zh-plugin-10.5.jar
+
 10.4.description=Support SonarQube 10.4
-10.4.sqVersions=[10.4,LATEST]
+10.4.sqVersions=10.4
```

Comment:
This should be `[10.4,10.4.1]` or `[10.4,10.4.*]`
If you don't leave this version marked compatible with 10.4.1, your users on that SQ version won't be prompted to upgrade to v10.4.1 of your plugin

Reply:
I changed sqVersion to [10.4,10.4.1]